src/NServiceBus.Core.Tests.x86/When_running_under_a_32Bit_process.cs

```diff
@@ -1,22 +0,0 @@
-﻿namespace NServiceBus.Core.Tests.x32
-{
-    using System.Linq;
-    using Hosting.Helpers;
-    using NUnit.Framework;
-
-    [TestFixture]
-    public class When_running_under_a_32Bit_process
-    {
-        [Test]
-        public void Should_skip_x64_assemblies()
```

Comment:
I guess as long as we have that scanner code we should keep the test. @Particular/nservicebus-maintainers I'll close this PR tomorrow unless someone thinks we should go ahead with this?